### DISCUSSION TITLE
Welcome to electrs Discussions!

### DISCUSSION THREAD

**Discussion by romanz** [romanz/electrs#350] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kixunil** [comment]
I love Bitcoin, Rust and Electrum, so `electrs` is my dream come true. :) I'm always looking for ways to improve `electrs` or user experience with Bitcoin in general. I have bunch of things in the pipeline in this area, but I need to focus a bit elsewhere for now.

**Comment by Alpa66alpa** [comment]
Hi, how can I install electrs on latest Mac OS? Any plan to cover the topic in details? 

Many thanks,
Al

**Reply by romanz** [reply]
Please try https://github.com/romanz/electrs/blob/master/doc/install.md - let us know if it works for you :)
